libs/utils-ts/src/VerifiedSale.ts

```diff
@@ -4,6 +4,10 @@ import { BuildVerifiedAction } from "./VerifiedAction";
 
 // ---------------------------------------------------------\\
 
+// We cannot use the following characters in the question/answer
+// Invalid characters: < or >, { or }, [ or ], %, &, #, \ or "
+export const invalidChars = /[\<\>\{\}\[\]\%\&\#\\\"]/g;
```

Comment:
Check was done server-side only before.  Seems a bit daft - bit late by the time hits our processor!